src/components/common/Button.js

```diff
@@ -0,0 +1,5 @@
+function Button({text}) {
+    return ( <button>{text}</button>);
```

Comment:
Not a big deal but I would remove the white space after the (

Reply:
done